docs/resources/key_pair.md

```diff
@@ -5,24 +5,26 @@ subcategory: ""
 description: |-
   Key pair resource schema. Must have a region specified in the provider configuration. Allows uploading an SSH public key to be used for server authentication.
   Usage with server
-  ```terraform
-  resource "stackitkeypair" "keypair" {
+  
```

Comment:
unrelated changes, not sure if they were missed in a previous PR or if you didn't merge with main

Reply:
This change was created by the doc generation. I'll inspect how this happened